packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewerContainer.js

```diff
@@ -94,42 +128,82 @@ class LightCurveViewerContainer extends Component {
   }
 
   render () {
-    const { subject } = this.props
+    const {
+      addAnnotation,
+      annotations,
+      currentTask,
+      drawFeedbackBrushes,
+      enableAnnotate,
+      enableMove,
+      feedback,
+      interactionMode,
+      onKeyDown,
+      setOnPan,
+      setOnZoom,
+      subject,
+      toolIndex,
+    } = this.props
+
     if (!subject.id) {
       return null
     }
 
     return (
       <LightCurveViewer
-        forwardRef={this.viewer}
+        addAnnotation={addAnnotation}
+        annotations={annotations}
+        currentTask={currentTask}
         dataExtent={this.state.dataExtent}
         dataPoints={this.state.dataPoints}
-        drawFeedbackBrushes={this.props.drawFeedbackBrushes}
-        feedback={this.props.feedback}
-        onKeyDown={this.props.onKeyDown}
+        drawFeedbackBrushes={drawFeedbackBrushes}
+        enableAnnotate={enableAnnotate}
+        enableMove={enableMove}
+        feedback={feedback}
+        forwardRef={this.viewer}
+        interactionMode={interactionMode}
+        onKeyDown={onKeyDown}
+        setOnPan={setOnPan}
+        setOnZoom={setOnZoom}
+        toolIndex={toolIndex}
       />
     )
   }
 }
 
-LightCurveViewerContainer.wrappedComponent.defaultProps = {
+LightCurveViewerContainer.defaultProps = {
+  addAnnotation: () => {},
+  drawFeedbackBrushes: () => {},
+  interactionMode: 'annotate',
   loadingState: asyncStates.initialized,
   onError: () => true,
   onReady: () => true,
   subject: {
     id: '',
     locations: []
-  }
+  },
 }
 
-LightCurveViewerContainer.wrappedComponent.propTypes = {
+LightCurveViewerContainer.propTypes = {
+  addAnnotation: PropTypes.func,
+  drawFeedbackBrushes: PropTypes.func,
+  interactionMode: PropTypes.oneOf(['annotate', 'move']),
   loadingState: PropTypes.string,
   onError: PropTypes.func,
+  onKeyDown: PropTypes.func.isRequired,
   onReady: PropTypes.func,
+  setOnPan: PropTypes.func.isRequired,
+  setOnZoom: PropTypes.func.isRequired,
```

Comment:
Why are these two callbacks required too? Can't they default to functions that return null or true?

Reply:
They didn't have defaults previously, so just being consistent about setting props that don't have defaults as required. It makes sense to me to have these be required later on when the view component itself gets put into its own library since the LCV functionally needs a zoom function to be set to be usable.